src/scripts/config.js

```diff
@@ -160,8 +164,14 @@
 
       // concept-coach
       OpenStaxConceptCoach: {
-        deps: ['css!../../bower_components/concept-coach/main.min'],
+        deps: ['css!../../bower_components/concept-coach/assets/main'],
```

Comment:
This change seems to be causing me to not be able to run `grunt` or `grunt dist`. Why was it changed? Reverting it to the old code seems to fix the problem for me.

Log output:

```
$ grunt dist

Running "requirejs:compile" (requirejs) task
>> Error: ENOENT: no such file or directory, open
>> '/path/to/webview-katalyst/dist/scripts/../../bower_components/
>> concept-coach/assets/main.css'
>> In module tree:
>>     css/css
Warning: RequireJS failed. Use --force to continue.

Aborted due to warnings.
$
```

A similar error occurs due to the change in line 33 of this file